### DISCUSSION TITLE
Add a metronome to detect note duration from midi controllers

### DISCUSSION THREAD

**Discussion by VaZark** [musescore/MuseScore#21493] [Ideas]
Just took a look at the video on how to use MIDI controllers with Musescore. While helpful, having to set the note duration for each input robs midi keyboards of their key feature - immersion. 

If Musescore can have a metronome and map the capture and release of keys to generate note durations in real time,  musescore can generate sheets for every session. It  would even help us see what notes we are hitting wrong or a random variation we came up with during practise & improvisation sessions

**Comment by MarcSabatella** [comment]
There already is a real-time metronome mode.  See the Handbook section on tentative note input methods.  And if you have further questions about how to use this feature, see the forums on musescore.org.  Fair warning: it's not a very sophisticated feature, and you'll find it's no magic bullet; there is no AI to sort out things like rhythms played imprecisely, use of multiple voices, use of multiple staves, etc.
